### ISSUE TITLE
Error in WinInc WINTRUST_FILE_INFO structure

### ISSUE THREAD

**Issue by vid512** [Terraspace/UASM#131]
WINTRUST_FILE_INFO.pgKnownSubject is supposed to be GUID*  (https://docs.microsoft.com/en-us/windows/win32/api/wintrust/ns-wintrust-wintrust_file_info)

WinTrust.INC in WinInc incorrectly declares it as:
```
pgKnownSubject	DWORD	?
```

**Comment by vid512** [comment]
Also in structure WINTRUST_DATA, all following should be PVOID, not DWORD:
```
pFile DWORD ?
pCatalog DWORD ?
pBlob DWORD ?
pSgnr DWORD ?
pCert DWORD ?
...
pwszURLReference        DWORD   ?
```

**Comment by john-terraspace** [comment]
Fixed for 2.10
